build/sections/qa.html

```diff
@@ -0,0 +1,49 @@
+<div class="container pt-28 mb-40">
+    <h3 class="text-3xl sm:text-2xl font-raleway font-bold mb-4">Fragen und Antworten</h3>
+    <p class="md:max-w-[55%] font-montserrat">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+        In consequat nec sapien ac sodales. Sed vitae urna nisi.
+        Morbi congue tincidunt porttitor. Aliquam sed venenatis 
+        quam, ac vulputate justo. Donec ullamcorper nunc quis sem 
+        bibendum, ut molestie lorem lobortis.
+    </p>
+
+    <div class="mt-8 mb-20 md:mb-44 space-y-10">
+        <div class="flex flex-col gap-y-12 sm:flex-row sm:justify-center gap-x-16 pt-4">
+            <div class="max-[374px]:w-full w-[340px] flex items-center border-b-4 border-primary-green pr-2">
+                <input class="qa-input" type="tel" name="" id="input-1" value="Welche Frage steht hier?">
+                <label for="input-1"><img class="cursor-pointer" width="11" src="Images/+.svg" alt=""></label>
+            </div>
+            <div class="max-[374px]:w-full w-[340px] flex items-center border-b-4 border-primary-green pr-2">
+                <input class="qa-input" type="tel" name="" id="input-2" value="Welche Frage steht hier?">
```

Comment:
This line contains `w-[340px]` which makes it not responsive because of the `px`. Change your width to `percentages`  or `rem`  they will be responsive